This patch is part of a series of patches for speech-tools for
Debian GNU/Linux, to provide Debian specific building of speech-tools

Index: b/config/config.in
===================================================================
--- a/config/config.in
+++ b/config/config.in
@@ -61,7 +61,7 @@
 # VERBOSE=1
 #DEBUG=1
 # PROFILE=gprof
-#SHARED=2
+SHARED=2
 
 ## Directory specific selections which override the above
 
